src/routes.js

```diff
@@ -660,6 +659,41 @@ const routes = [
     }
   }),
 
+  asyncRoute({
+    path: '/projects/:projectId([1-9]\\d*)/forms/:xmlFormId/submissions/:instanceId/edit',
+    component: 'FormSubmission',
+    name: 'WebFormEditSubmission',
+    props: true,
+    loading: 'page',
+    meta: {
+      standalone: true,
+      title: () => [form.nameOrId],
+    }
+  }),
+  asyncRoute({
+    path: '/projects/:projectId([1-9]\\d*)/forms/:xmlFormId/submissions/new',
```

Comment:
Wouldn't a path that matches this route also match `SubmissionShow`? Technically, `new` is a valid instance ID. I'm not too worried about someone using an instance ID of `new`, but I'm confused about how this is working exactly.